app/src/main/java/com/infomaniak/mail/utils/UiUtils.kt

```diff
@@ -38,13 +38,13 @@ object UiUtils {
     fun formatUnreadCount(unread: Int) = if (unread >= 100) "99+" else unread.toString()
 
     fun fillInUserNameAndEmail(
```

Comment:
Maybe move this to Correspondant interface to use it as an extension function ?

Reply:
🙅

Reply:
Because this modifies the UI, this won't go in Correspondent